Scripts/script-ReadFile.yml

```diff
@@ -10,7 +10,10 @@ script: |-
       maxFileSize = 1024**2
   res = demisto.executeCommand('getFilePath', {'id': demisto.args()['entryID']})
   with open(res[0]['Contents']['path'], 'r') as f:
-      data = f.read(maxFileSize).decode('unicode_escape').encode('utf-8')
```

Comment:
Use: `errors='replace'` for the decode function

Reply:
What do you mean by `errors='replace'`?

Reply:
`decode` method takes as second parameter an `errors` argument. It is best to use `replace` for the `errors` arg.